user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -205,6 +205,35 @@ const handler = getLambdaHandler(myContract)(async event => {
 });
 ```
 
+You can also pass additional arguments to the handler, but keep in mind that you will need to provide default values as Lambda will not pass them for you. Also bear in mind that lambda passes `context` and `callback` to your lambda, so any additional argument **MUST** be after. See https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html.
+
+```ts
+interface AdditionalArgs {
+  toto: string;
+}
+
+const additionalArgs = {
+  toto: 'tata',
+};
+
+const handler = getLambdaHandler(myContract)(
+  async (
+    event, // type will be properly inferred
+    _context, // be careful to define these arguments even if you don't use them!
+    _callback,
+    myAdditionalArg: AdditionalArgs = additionalArgs, // here you need to manually define a default
+  ) => {
```

Comment:
I see you point, maybe it is overkill to document it. In any case, access to `context` and `callback` should be allowed as they may be used by the lambda.

I'll simply remove the section in the docs

Reply:
It's ok for me!
One last thing, in the [docs](https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html#non-async-typescript), it's written that the third argument `callback` is only present when the handler is non-async. And that it is not present when the handler is async.
Since our handlers are async, the `callback` parameter should not work.

Reply:
I checked it and the `callback` argument is also passed to async lambdas, it's just completely useless I think